app/models/vmdb_database_setting.rb

```diff
@@ -20,25 +20,13 @@ def vmdb_database
 
   delegate :id, :to => :vmdb_database, :prefix => true
 
-  def minimum_value
-    min_val || ''
-  end
-
-  def maximum_value
-    max_val || ''
-  end
-
-  def value
-    setting || ''
-  end
+  alias_attribute :minimum_value, :min_val
```

Comment:
removed these changes.
Do wish we were using `alias_attributes`, but c'est la vie.